src/worker/index.js

```diff
@@ -8,3 +8,7 @@ testplaneFacade.init();
 exports.runTest = (fullTitle, options) => {
     return testplaneFacade.runTest(fullTitle, options);
 };
+
+exports.cancel = () => {
```

Comment:
I can't stop repl server from master process. So I need to send command to all workers.